### PR TITLE
April 2024 newsletter

### PR DESCRIPTION
**Pull request by davidfischer**
I treated this as a bit of a soft-launch for the niche targeting. Also, I wasn't sure how much detail about softness in sales or whether we should discuss any sort of retention discounts.

### CODE REVIEW

File: content/posts/newsletter-april-2024.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+  with simplified graphics and explainers about what we expect from publishers
+  and how they can get the most out of EthicalAds.
+* We released a new [advertiser pricing page]({filename}../pages/advertisers-pricing.md).
+  While our pricing was always upfront, we think having a dedicated pricing page
+  in our core navigation will help us set the tone for how we're upfront and transparent with advertisers.
+* We're continuing to make progress on our
+  [machine learning based niche targeting]({filename}../posts/2022-content-based-targeting.md).
+  These changes will let us target a lot of smaller audiences that were previously too hard or too small to target.
+  We're going to do a bigger launch with blogs and social media explaining this change in April,
+  but you can see a technology demo where we show the ability to find similar pages to an advertiser's product or landing page [here]({filename}../pages/similar-pages.md).
```
Comment (ericholscher) [inline content/posts/newsletter-april-2024.md:22]: Should we link to an example with a URL filled?

File: content/posts/newsletter-april-2024.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+In March 2024:
+
+* We generated **$44,375.69** for our publishers.
+* We had **177 publishers** on our network with at least one paid ad impression.
+* We had **17,592,121** paid ad views across the world.
+
+March was a very strong month but we're seeing a little softness in early April.
+Some of this may be seasonality but we're seeing some established advertisers
+pulling back in April.
```
Comment (ericholscher) [inline content/posts/newsletter-april-2024.md:42]: Maybe worth noting we're thinking about doing some loyalty discounts, but yea not sure how to frame it publicly, so maybe best to just keep it to email for now.

**Review by ericholscher — APPROVED** [review]
Looks good 👍

File: content/posts/newsletter-april-2024.md (hunk 1 above)
Comment (davidfischer) [reply content/posts/newsletter-april-2024.md:22]: Seems reasonable.

**Comment by davidfischer** [comment]
I pushed an update that links to a specific URL. I think the main thing I'd like to get out tomorrow around the time this releases is an update to limit the distance on the results. I have it mostly running but I'm struggling with some environment issues with the squash migration.

**Pull request merged**